cli/command/network/create.go

```diff
@@ -63,6 +66,12 @@ func newCreateCommand(dockerCli *command.DockerCli) *cobra.Command {
 	flags.SetAnnotation("attachable", "version", []string{"1.25"})
 	flags.BoolVar(&opts.ingress, "ingress", false, "Create swarm routing-mesh network")
 	flags.SetAnnotation("ingress", "version", []string{"1.29"})
+	flags.BoolVar(&opts.swarm, "swarm", false, "Create a swarm network")
```

Comment:
Yeah, we are still debating internally about the proper flag. 
We are thinking of `--scope=[local|swarm]` where if not specified, the default network driver scope is used. So the flag would be to promote to `swarm` scope a network which default driver socpe is `local`. It seems like there would be future use-cases for the reverse, the downgrade.